frameworks/PHP/symfony/src/Entity/World.php

```diff
@@ -10,59 +10,37 @@
  */
 class World
 {
-   /**
-    * @ORM\Id
-    * @ORM\Column(type="integer")
-    */
+    /**
+     * @ORM\Id
+     * @ORM\Column(type="integer")
+     */
     public $id;
 
-   /**
-    * @ORM\Column(type="integer")
-    */
-    public $randomNumber;
-
     /**
-     * Set id
-     *
-     * @param integer $id
-     * @return World
+     * @ORM\Column(type="integer")
      */
-    public function setId($id)
+    public $randomNumber;
+
+    public function setId(int $id): World
     {
         $this->id = $id;
-    
+
         return $this;
     }
 
-    /**
-     * Get id
-     *
-     * @return integer 
-     */
-    public function getId()
+    public function getId(): int
     {
         return $this->id;
     }
 
-    /**
-     * Set randomNumber
-     *
-     * @param integer $randomNumber
-     * @return World
-     */
-    public function setRandomNumber($randomNumber)
+    public function setRandomNumber(int $randomNumber): World
```

Comment:
does not look like we need to make this method chainable.

Reply:
i agree - not a fan of 'fluid' methods neither.
i can change this.